packages/usa-header/src/styles/_usa-megamenu.scss

```diff
@@ -15,20 +15,17 @@
     $mw: units($header-max-width);
   }
 
-  // Calculate the width of the outer megamenu
-  // Subtract half the value of the viewport width from half the value of the submenu width
-  //
-  // Standard needs the additional $theme-site-margins-width to accommodate
-  // different paddings/structure on #basic-mega-nav-section-two
-  @if $type == "extended" and $header-max-width == "none" {
-    width: 0;
+  // Create the vw equivalent for max-width:none
+  @if $header-max-width == "none" {
+    $mw: 100vw;
+  }
+
+  // subtract half the value of the viewport width from half the value of the submenu width
+  // standard needs the additional $theme-site-margins-width to accommodate different paddings/structure on #basic-mega-nav-section-two
```

Comment:
Does this comment make sense here or should we move it inside? 

Let's break up long comments to next line to keep these comments readable. In my editor they look like this:

![image](https://user-images.githubusercontent.com/3385219/216109476-464b2ef2-563e-47d9-bb81-32d51ca73632.png)


```suggestion
  // Standard needs the additional $theme-site-margins-width to 
  // accommodate different paddings/structure on Megamenu variant.
```